### DISCUSSION TITLE
Automatic List and Header generation, like in Dropbox Paper, Google Docs or Evernote?

### DISCUSSION THREAD

**Discussion by katerlouis** [tinymce/tinymce#8376] [General]
In Dropbox Paper, when you type `* `  or `- ` (asterisk/minus and space) it automatically transforms to a list tool. Analog to this `### ` would transform to a Header level 3 tool.

Is this possible with tinymce aswell?
I'm surprised I haven't found anything in the docs or questions

**Comment by TheSpyder** [comment]
It was an optional feature in TinyMCE 5, on by default in TinyMCE 6. You perhaps couldn't find it because it's under "text patterns".
https://www.tiny.cloud/docs/tinymce/6/content-behavior-options/#text_patterns

**Reply by katerlouis** [reply]
That's what I'm looking for, great!
I'm wondering though; for me it is not on by default in the homepages demo: https://www.tiny.cloud/
